README.md

```diff
@@ -11,7 +11,7 @@ Command Line Argument Parser for Rust
 It is a simple-to-use, efficient, and full-featured library for parsing command line arguments and subcommands when writing console/terminal applications.
 
 * [Documentation](https://docs.rs/clap/)
-* [Chat, questions go there](https://gitter.im/kbknapp/clap-rs)
+* [Discussions](https://github.com/clap-rs/clap/discussions)
```

Comment:
I think we need to explicitly highlight that questions should be directed to discussions.